templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -13,7 +13,8 @@ contents:
     spec:
       initContainers:
       - name: discovery
-        image: "{{.Images.setupEtcdEnv}}"
+        image: "{{.Images.machineConfigOperator}}"
```

Comment:
Ok, I see it 😄 we still have to use `Images. setupEtcdEnv`cause that's the key being substituted

Reply:
right, I've created a smoke PR (now closed) https://github.com/openshift/machine-config-operator/pull/915 and it shows tests passing

Reply:
i stared at this for so long!!!! can't believe i didnt see this - thank you!